docs/api-usage/authentication.mdx

```diff
@@ -105,9 +114,53 @@ You can also use the [`me`](api-reference/queries/me.mdx) query to retrieve info
 }
 ```
 
+### Verifying tokens
+
+:::tip
+When implementing a storefront, you should probably use the [`me`](#fetching-information-about-currently-authenticated-user) query instead.
```

Comment:
The sentence says: "you should probably use the `me` query instead", but it's unclear to me what we should use it for.

We could use some context for why these two ("verifying tokens" and "me query") are related to each other. 

I think the structure of the article is the source of my confusion. From such a flat structure, it's difficult to tell relationships between sections:

<img width="236" alt="image" src="https://user-images.githubusercontent.com/44495184/216537732-11b68063-b23d-4f32-bde2-92490d22df2d.png">

Maybe those two paragraphs should be nested under one heading?

If you have any idea for how we can better structure this whole document, feel free to suggest it, and I can implement it in the next PR.